node/tests/GlideClient.test.ts

```diff
@@ -481,22 +481,22 @@ describe("GlideClient", () => {
             );
 
             const result = await client.lolwut();
-            expect(result).toEqual(expect.stringContaining("Redis ver. "));
+            expect(result).toEqual(expect.stringContaining("Valkey ver. "));
```

Comment:
this will fail

Reply:
Correct. The output is still "Redis ver. ". Please double check.